src/main/java/ru/yandex/practicum/filmorate/model/Film.java

```diff
@@ -26,15 +26,11 @@ public class Film {
     private LocalDate releaseDate;
     @Positive
     private int duration;
-    private List<Genres> genres;
-    private Ratings rating;
-    private final Set<Integer> likes = new HashSet<>();
+    private List<Genres> genres = new ArrayList<>();
```

Comment:
Инициализация лишняя, мы инициализируем это поле через конструктор, который генерирует lombok

```suggestion
    private List<Genres> genres;
```